libs/shared/src/types/feature-flags/feature-flags.ts

```diff
@@ -1,10 +1,10 @@
 export enum FeatureFlagsKeysEnum {
   IS_TEMPLATE_STORE_ENABLED = 'IS_TEMPLATE_STORE_ENABLED',
-  IS_TOPIC_NOTIFICATION_ENABLED = 'IS_TOPIC_NOTIFICATION_ENABLED',
```

Comment:
This one was an easy cleanup. It's toggled true in all environments for 8 months, and can now be deleted.

Reply:
This is one layer I wonder what the launch darkly value is and if we don't need this feature flag anymore.
Removing it in this pull request might be a straightforward improvement, but I'm wondering whether we should establish a call to action (CTA) to remove feature flags with a dedicated ticket.

On a side note, if we had a specific ticket template for removing a feature flag, we could include some clear guidelines in DODs to ensure we don't forget sub-tasks. For example, one of them could be to confirm the removal of the flag in LaunchDarkly so that we don't end up with any outdated flags there.

Reply:
Good point. I will reverse the removal of this FF and I agree we can handle it separately.